index.html

```diff
@@ -0,0 +1,116 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title> Build A Survey Form</title>
+        <!-- Added css file  -->
+        <link rel="stylesheet" href="style.css">
+    </head>
+    <body>
+        <div class="container">
+
+            <header class="header">
+                <h1 id="title" class="text-center">Microvarse Survey Form</h1>
+                <p id="description" class="description text-center">
+                    Thanks for taking the time to help us improve the platform
+                </p>
+            </header>
+            <form id="survey-form">
+                <div class="form-group">
+                    <label id="name-label" for="name">Name</label>
+                    <input type="text" name="name" id="name" class="form-control" placeholder="Enter your name" required />
+                </div>
+                <div class="form-group">
+                    <label id="email-label" for="email">Email</label>
+                    <input type="email" name="email" id="email" class="form-control" placeholder="Enter your Email" required />
+                </div>
+
+
+        <div class="form-group">
+            <label id="number-label" for="number">Age<span class="clue">(optional)</span></label>
+            <input type="number" name="age" id="number" min="10" max="99" class="form-control" placeholder="Age" />
+        </div>
+        <div class="form-group">
+            <p>Which option best describes your current role?</p>
+            <select id="dropdown" name="role" class="form-control" required>
+                <option disabled selected value>Select current role</option>
+                <option value="student">Student</option>
+                <option value="job">Full Time Job</option>
+                <option value="learner">Full Time Learner</option>
+                <option value="preferNo">Prefer not to say</option>
+                <option value="other">Other</option>
+            </select>
+</div>
+<div class="form-group">
+    <p>Would you recommend Microvarse to a friend?</p>
+    <label>
+        <input name="user-recommend" value="definitely" type="radio" class="input-radio" checked />Definitely</label>
+    <label>
+        <input name="user-recommend" value="maybe" type="radio" class="input-radio" />Maybe</label>
+
+    <label><input name="user-recommend" value="not-sure" type="radio" class="input-radio" />Not sure</label>
```

Comment:
Please always use the `label` tag with `for` attributes, The `for` attribute specifies which form element a label is bound to.

Reply:
You have given reasons why the user should make the required change but I think we can also improve this comment by being more suggestive rather than instructive or giving orders😉😉 kindly refer back to the first comment for a clarity🤗